File: C_Codes/Ponteiros/pointer.c
#include <stdio.h>

int main (){

    int var = 10;
    int *ptr;
    ptr = &var;

    printf("Valor de var: %d\n", *ptr); // Desreferencia o ponteiro para acessar o valor de var
    *ptr = 20; // Modifica o valor de var através do ponteiro
    printf("Novo valor de var: %d\n", var);


}

File: Códigos/Python Codes/Ponteiros/pointer.py
def main():
    var = 10
    # Em Python, não usamos ponteiros. Podemos manipular a variável diretamente.
    ptr = var

    print(f"Valor de var: {ptr}")  # Acessa o valor diretamente
    ptr = 20  # Modifica o valor de var através da variável
    print(f"Novo valor de var: {ptr}")


if __name__ == "__main__":
    main()
